### DISCUSSION TITLE
Partial Fourier Data

### DISCUSSION THREAD

**Discussion by tknopp** [MagneticResonanceImaging/MRIReco.jl#164] [General]
Hi @aTrotier,

I was wondering, if you did measure some partial Fourier data with a Bruker scanner, so that we can try reconstructing it. I have seen, that you included functions to read out the information, but if I understand correctly, we can't reconstruct right now.

**Comment by aTrotier** [comment]
I might have that somewhere or I can acquire a new dataset according to your needs.

I think the reconstruction works (CG-sense or CS) but we don't use the symmetry property of k-space.

Do you want to implement a more simple PF algorithm (homodyne / POCS) ?

**Reply by tknopp** [reply]
yes, we want to implement a simple direct reconstruction. But at the end I am also interested if there are any opportunities in CS based reconstruction when using partial Fourier. I simply do not know.

**Reply by tknopp** [reply]
@migrosser

**Reply by aTrotier** [reply]
The POCS algorithm is the most common one used on all the scanner. Not sure if we should let the user pass the low-res phase as in params["phase_map"] or directly estimate it in the reconstruction but we have less control over the smoothing/edge transition...

Anyway, maybe we should take a look about using the MRD label for calibration lines

**Reply by aTrotier** [reply]
Do you need a PV6.0.1, PV360.3.5 dataset or both ?

**Reply by tknopp** [reply]
I think the new PV version would be sufficient. Important would be to include a reconstruction, so that we can compare.

**Comment by aTrotier** [comment]
I think in pv6.0.1, the partial Fourier undersampling was only
reconstructed with zero filling algorithm.

Now pocs is available. I'll acquire a dataset next week.

Le ven. 1 déc. 2023 à 21:26, Tobias Knopp ***@***.***> a
écrit :

**Reply by tknopp** [reply]
interesting. So do you have both versions installed and could potentially convert? If that is not to much of an effort, both versions would of course be helpful.

**Comment by aTrotier** [comment]
We have a 4.7T with pv6.0.1 and a 7t with pv360. I don't think I can
transfert dataset between both version.

Le sam. 2 déc. 2023 à 09:07, Tobias Knopp ***@***.***> a
écrit :

**Comment by aTrotier** [comment]
The bruker shared dataset has a PF acquisition reconstructed with POCS I think
